themes/digital.gov/src/scss/new/guides/_guide-menu-bar.scss

```diff
@@ -0,0 +1,63 @@
+@use "uswds-core" as *;
+
+.dg-guide__menu-bar {
+  @include u-border-y("1px", "gray-cool-30", "solid");
+  @include u-z("top");
+  align-items: center;
+  background-color: color("gray-10");
+  display: flex;
+  height: units(10);
+  justify-content: center;
+  position: sticky;
+  top: -1px;
+  white-space: nowrap;
+
+  &-image {
+    display: none;
+    margin: 0 units(3);
+    &.full {
+      min-width: units("card");
+      width: units("card");
+    }
+    &.mobile {
+      width: units(3);
+    }
+    &.sticky {
+      display: inline;
+    }
+  }
+
+  &-link {
+    @include u-font("sans", "md");
+    @include u-text("light", "black");
+    margin: 0 units(3);
+    text-decoration: none;
+
+    &:hover {
+      @include u-border-bottom("05", "primary", "solid");
+      color: color("black");
+    }
+    &:visited {
+      color: color("black");
+    }
+  }
+
+  &-links {
+    -webkit-mask: linear-gradient(90deg, #0000, #000 5% 95%, #0900);
+    mask: linear-gradient(90deg, #0000, #000 5% 95%, #0000);
+    -ms-overflow-style: none; //Internet Explorer 10
```

Comment:
There _should_ be autoprefixer enabled that will set prefixes for us. If it isn't we should create an issue to investigate why.

We're not supporting IE10, so safe to remove.